src/reference/torii-endpoints.md

```diff
@@ -164,7 +198,10 @@ The progress on the configuration reference can be tracked in the following GitH
 
 ::: tip Guarantees
 
-A successful configuration update does not guarantee that the configuration is indeed updated. While a follow-up [Configuration / Retrieve](#configuration-retrieve) request will return updated values, the actual update is performed asynchronously.
+A successful configuration update does not guarantee that the configuration is
+indeed updated. While a follow-up
+[Configuration / Retrieve](#configuration-retrieve) request will return updated
+values, the actual update is performed asynchronously.
```

Comment:
This tip is a bit confusing. Perhaps an explanation of how to determine that the configuration update was in fact completed successfully should be added?

Reply:
I would say it isn't really necessary. Maybe we can omit this tip entirely.

Yes, it could fail, but in a very rare case. And log error will be printed.
Whoever tries to update the configuration will see the error anyway.

Reply:
Removed